mesonbuild/interpreter.py

```diff
@@ -981,6 +981,8 @@ def method_call(self, method_name, args, kwargs):
             fn = getattr(self.held_object, method_name)
         except AttributeError:
             raise InvalidArguments('Module %s does not have method %s.' % (self.modname, method_name))
+        if method_name.startswith('_'):
+            raise InvalidArguments('Function %s in %s module is private.' %(method_name, self.modname))
```

Comment:
Please use `{!r}` + `.format()` here:

```
InvalidArguments('Function {!r} in module {!r} is private.'.format(method_name, self.modname))
```

I think we should use `.format()` everywhere consistently and change the surrounding code too when we touch it.